### DISCUSSION TITLE
Why does create a custom connection not take effect

### DISCUSSION THREAD

**Discussion by bjfk2006** [RamiAwar/dataline#251] [Q&A] [answered]
![image](https://github.com/RamiAwar/dataline/assets/6290119/de79ca77-b67e-4f2c-b53c-c86af49cb0dd)
DSN correct format?

**Comment by RamiAwar** [comment, marked as answer]
DSN looks good, I think you're missing a name on the top of the page (outside screenshot) 😄 

Does that fix it?

All good feedback tho, maybe we should just provide a default name
